docs/sqlite-sync.md

```diff
@@ -0,0 +1,98 @@
+---
+id: sqlite
+title: SQLite Sync
+sidebar_label: API
+---
+
+The `@databases/sqlite-sync` library provides a _synchronous_, safe and convenient API
+for querying SQLite databases in node.js. Built on top of
+[better-sqlite3](https://www.npmjs.com/package/better-sqlite3).
+
+N.B. you should only have one process connected to a given SQLite database at a time.
+
+## Usage
+
+```typescript
+import connect, {sql} from '@databases/sqlite-sync';
+// or in CommonJS:
+// const connect = require('@databases/sqlite-sync');
+// const {sql} = require('@databases/sqlite-sync');
+
+const db = connect();
+
+console.log(db.query(sql`SELECT * FROM users;`));
+```
+
+```javascript
+const connect = require('@databases/sqlite-sync');
+const {sql} = require('@databases/sqlite-sync');
+
+const db = connect();
+
+console.log(db.query(sql`SELECT * FROM users;`))
+```
+
+> For details on how to build queries, see [Building SQL Queries](sql.md)
+
+## API
+
+### `connect(fileName)`
+
+Create a database connection for a given database. You should only create one connection per database for your entire applicaiton. Normally this means having one module that creates and exports the connection pool.
+
+In memory:
+
+```ts
+import connect from '@databases/sqlite';
+const db = connect();
+```
+
+File system:
+
+```ts
+import connect from '@databases/sqlite';
+const db = connect(FILE_NAME);
+```
+
+The `Database` inherits from `DatabaseTransaction`, so you call `Database.query` directly instead of having to create a transaction for every query. Since SQLite has very limited support for actual transactions, we only support running one transaction at a time, but multiple queries can be run in parallel. You should therefore only use transactions when you actually need them.
```

Comment:
I think since the entire API is synchronous, we can remove the entire sentence about lack of support for concurrent transactions.

```suggestion
The `Database` inherits from `DatabaseTransaction`, so you call `Database.query` directly instead of having to create a transaction for every query.
```